webpack.prod.config.babel.js

```diff
@@ -30,9 +30,9 @@ export default {
     filename: '[name]-[contenthash].js',
     chunkFilename: '[name]-[contenthash].js',
     // This is the path used to require files in the generated bundles.
-    publicPath: `${
-      config.has('staticHost') ? config.get('staticHost') : ''
-    }/static/`,
+    publicPath: `${config.has('staticHost') ? config.get('staticHost') : ''}${
+      config.has('staticPath') ? config.get('staticPath') : '/static/'
```

Comment:
I suggest we make `staticPath` a required config parameter, with `/static/` being the default value.

Reply:
Also, would it be a problem to rename this folder in all envs?

Reply:
`/static-frontend/` could technically work if the ops side is deployed to stage & prod but this isn't the case yet so I didn't want to use it everywhere yet.

Reply:
I am thinking that it might make things simpler to land a patch with just the static path change and then this patch for the same domain on -dev, but up to you really.

Reply:
https://github.com/mozilla/addons-frontend/commit/949699383b1025e9641528792ae51eb65c30b398 makes us use `/static-frontend/` for everything.